apps/web/package.json

```diff
@@ -97,6 +97,7 @@
     "jest-fail-on-console": "^3.3.0",
     "prettier": "^3.3.3",
     "ts-jest": "^29.2.3",
+    "ts-node": "^10.9.2",
```

Comment:
I don't see `ts-node` imported anywhere: what is it used by and what for?

Reply:
Oh ya sorry,  this is required to read typescript config files.

"To read TypeScript configuration files Jest requires [ts-node](https://npmjs.com/package/ts-node). Make sure it is installed in your project." - [Jest Docs](https://jestjs.io/docs/configuration)